yarn.lock

```diff
@@ -1,2747 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Can you regenerate this lock file please

Reply:
lock files are not recommended for libraries, see my comment below about package-lock.json and maybe it applies to yarn as well, I don't actually know much about yarn tbh

Reply:
The lockfile is still useful for anyone developing on this project.